### ISSUE TITLE
[BUG]: Cannot open sound config dialog in v12

### ISSUE THREAD

**Issue by TaureHorn** [Aedif/token-sounds#6]
Clicking on the '+' icon in the token sounds submenu of a token doesn't open the sound config window in foundry v12. Build 327.

Checked in two game systems (DnD5e v3.0.1 and CY_BORG) just in case and issue persists on both.

**Comment by Fridan99** [comment]
Any plans to upgrade to v12?

**Comment by Aedif** [comment]
Yup, most of the work has already been done for it. The release will likely be pushed out over the weekend.

**Comment by Fridan99** [comment]
Great :), thanks for the work
